pkg/certs/module.go

```diff
@@ -129,13 +133,19 @@ func (a *AutoRenewCertsModule) Init() {
 	a.Name = "AutoRenewCertsModule"
 	a.Desc = "Install auto renew control-plane certs"
 
+	containerManager := a.KubeConf.Cluster.Kubernetes.ContainerManager
+	kubeadmVersionCmp, _ := versionutil.MustParseSemantic(a.KubeConf.Cluster.Kubernetes.Version).Compare("v1.20.0")
 	generateK8sCertsRenewScript := &task.RemoteTask{
 		Name:  "GenerateK8sCertsRenewScript",
 		Desc:  "Generate k8s certs renew script",
 		Hosts: a.Runtime.GetHostsByRole(common.Master),
 		Action: &action.Template{
 			Template: templates.K8sCertsRenewScript,
 			Dst:      filepath.Join("/usr/local/bin/kube-scripts/", templates.K8sCertsRenewScript.Name()),
+			Data: util.Data{
+				"IsDocker":            (containerManager == common.Docker),
+				"IsKubeadmAlphaCerts": (kubeadmVersionCmp == -1),
```

Comment:
May be can use:
```
"IsKubeadmAlphaCerts": versionutil.MustParseSemantic(a.KubeConf.Cluster.Kubernetes.Version).LessThan("v1.20.0")
```
This method will return a bool value. And looks more intuitive.